### PR TITLE
Wrong birthdate format (EXPOSUREAPP-8283)

### PR DESCRIPTION
**Pull request by mtwalli**
- Format birthdate as `dd.mm.yyyy` if possible 😐

### CODE REVIEW

**Comment by vaubaehn** [comment]
@mtwalli 
Firstname and Surname are also swapped - is it also on your list or should I create an issue?

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/CwaUserCard.kt [hunk 1]
```diff
@@ -38,11 +42,23 @@ class CwaUserCard(parent: ViewGroup) :
         }
     }
 
+    private fun formatBirthDate(dateOfBirthFormatted: String): String =
+        try {
+            LocalDate.parse(dateOfBirthFormatted, format).toDayFormat()
+        } catch (e: Exception) {
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/CwaUserCard.kt:48]: This does not work for dates with only a year or a year and month as it add the missing values

**Review by chiljamgossow — CHANGES_REQUESTED** [review]
Does not work for YYYY-MM or YYYY

**Comment by mtwalli** [comment]
> @mtwalli
> Firstname and Surname are also swapped - is it also on your list or should I create an issue?
> Edit: in vacc details screen

Please another ticket ? this PR is addressing date only

**Comment by vaubaehn** [comment]
> Please another ticket ? this PR is addressing date only

Sure, just didn't want to waste time in case it's already on your list.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/CwaUserCard.kt (hunk 1 above)
Comment (mtwalli) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/CwaUserCard.kt:45]: Fixed

**Review by chiljamgossow — APPROVED** [review]

**Review by BMItr — APPROVED** [review]
lgtm

**Pull request merged**